tests/unit/SidebarFilterTest.js

```diff
@@ -10,6 +10,7 @@ import wrapOnyxWithWaitForBatchedUpdates from '../utils/wrapOnyxWithWaitForBatch
 
 // Be sure to include the mocked permissions library or else the beta tests won't work
 jest.mock('../../src/libs/Permissions');
+jest.mock('../../src/hooks/usePermissions.ts');
```

Comment:
Yeah I can update the comment. Some of the parent components are using `Permissions.canDoSomething` and I used `usePermissions` at the request of the other reviewer. It seemed out of scope to change to update the other parent components to using the hook, but wouldn't be difficult. I did include this mock because the tests were failing otherwise.